WordPress/Classes/Services/ReaderPostService.m

```diff
@@ -930,8 +935,8 @@ - (void)deletePostsForTopic:(ReaderAbstractTopic *)topic missingFromBatch:(NSArr
             continue;
         }
         // The post was missing from the batch and needs to be cleaned up.
-        if (post.inUse) {
-            // If the missing post is currenty being used just remove its topic.
+        if (post.inUse || post.isSavedForLater) {
```

Comment:
We are doing this boolean check at least three times, would it might make sense to encapsulate it in a method that will not only remove that duplication but, as an added bonus, would make this point more readable? I was thinking something along the lines of

``
if ([self topicMustBeNullifiedForPost: post]
``

(I know the name is terrible, but I hope it illustrates my point)